backend/routes/webAuthn.go

```diff
@@ -13,6 +13,7 @@ func WebAuthnRoute() chi.Router {
 	r.Post("/authorize/request", controller.AuthorizeRequest)
 	r.Post("/authorize/response/{doorId}/{challenge}", controller.AuthorizeResponse)
 	r.Get("/authorize/{doorId}/{yubiKeyId}", controller.GetYubiKeyDoorAccess)
+	r.Delete("/sessionData", controller.DeleteSessionData)
```

Comment:
@cameronkinsella should we put this in admin only routes?

Reply:
Yeah. I'd prefer if we could add it under one of the base paths that are already private. All other endpoints in the /auth path are already public.

Reply:
Yeah, it's hard to say. I would be fine with adding a new /creds path. You could set endpoint as `/` for that path (i.e. `DELETE /creds`).
Thoughts? @AdrianJendo

Reply:
That sounds good to me